x.toml

```diff
@@ -256,12 +257,6 @@ existing_deps = [
     # Tier 0 - essential components that must be included in the first batch of crates getting
     #          moved to the new repo
     ["move-lang", "diem-framework"],                          # sanity check to ensure that the diem-framework compiles
-    ["compiler", "diem-framework-releases"],
```

Comment:
I forgot to delete this in #8576. Doing it here just to save myself from some more merge conflicts